src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/ScheduleResourceTest.kt

```diff
@@ -365,6 +365,25 @@ class ScheduleResourceTest : ResourceTest() {
     }
   }
 
+  @Nested
+  @DisplayName("GET /api/schedules/{prisonerNumber}/scheduledTransfers")
+  inner class ScheduledTransfers {
+
+    @Test
+    fun testThatScheduledTransfer_IsReturned() {
+      val token = authTokenHelper.getToken(NORMAL_USER)
+      val response: ResponseEntity<List<PrisonerSchedule>> = testRestTemplate.exchange(
```

Comment:
prefer web test client over deprecated testRestTemplate - but I understand if you chosen this to be consistent with the other tests for this resource